core/domain/topic_domain.py

```diff
@@ -1187,8 +1191,7 @@ def _convert_subtopic_v3_dict_to_v4_dict(
         Returns:
             dict. The converted subtopic_dict.
         """
-        fs = fs_domain.AbstractFileSystem(fs_domain.GcsFileSystem(  # type: ignore[no-untyped-call]
-            feconf.ENTITY_TYPE_TOPIC, topic_id))
+        fs = fs_services.GcsFileSystem(feconf.ENTITY_TYPE_TOPIC, topic_id) # type: ignore[no-untyped-call]
```

Comment:
Is this ignore needed?

Reply:
Yes it required because _topic_domain.py_ is type annotated and _fs_services. is not.